signin.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>vaspacx/login</title>
+    <link rel="stylesheet" href="login.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <div class="login-box">
+
+
+  <div class="login-logo-container">
+    <span class="vaspacx">vaspac<i class="fa fa-xing" aria-hidden="true"></i></span>
+    <div class="social-logo-cont">
+      <i class="fa fa-github" aria-hidden="true"></i>
+      <i class="fa fa-twitter" aria-hidden="true"></i>
+      <i class="fa fa-facebook" aria-hidden="true"></i>
+    </div>
+    <a href="https://vaspacx-ecommerce.netlify.app/mainpage.html"><button class="primay-btn outl-btn">Shop now </button></a>
+  </div>  
+    
+    <div class="astronaut-cont">
+        <img src="./img/Astronaut-rafiki.png">
+    </div>
+    <div class="container-login">
+        <div class="cont-left">
+            <h1>Hi, Welcome Back!</h1>
+            <div>
+                <small class="text-grey">Don't have an account ?<a href="signup.html"> Sign up</a></small>
+            </div>
+            <label>E-mail</label><br>
+            <input id="input" type="email" required placeholder="E-mail">
+            <label>Password</label><br>
+            <input id="input" type="email" required placeholder="Password">
+            <small> <u class="text-grey pointer">Forgot Password?</u></small>
+            <div class="btn-container-login"><button class="btn-create">Sign In</button>
+            <button class="btn-sign"><i class="fa fa-google-plus-square" aria-hidden="true"></i>Sign in with Google</button>
+            </div>
+            <small class="text-grey">@vaspacx-Privacy Policy and Terms of Service apply.</small>
+          </div>
+         
+    </div>
+    <img src="./img/sun.png" id="icon-login">
+  <!-- </div> -->
+</div>
+  <script>
+    var icon = document.getElementById("icon-login");
+
+    icon.onclick = function(){
+      document.body.classList.toggle("light-theme");
+      if(document.body.classList.contains("light-theme")){
+        icon.src="./img/moon.png";
+      }else{
+        icon.src = "./img/sun.png"
+      }
+    }
```

Comment:
its better to create a different JS file rather than writing js in html itself

Reply:
it's few line of code that'y I put it in same file if the code line increases I will definitely put it in another file